### DISCUSSION TITLE
how to deposit a script event file function truffle suite

### DISCUSSION THREAD

**Discussion by Fracs99** [ConsenSys-archive/truffle#5154] [General Q&A]
**In the test folder of my truffle project I go to test the smart contracts, how can I deposit an integer by calling the function present in the smart contract without deploying the contract again?

**Comment by apdan510** [comment]
@Fracs99 Thanks for reaching out. Could you please provide an example or a snippet of your code? Are you attempting to pull contract data and modify it in your frontend when it is called? 

Kind regards,
Alex | Truffle Support
